rss_controller.py

```
@@ -0,0 +1,47 @@
+import sqlite3
```

Ruff (0.11.9)

`sqlite3` imported but unused

Remove unused import: `sqlite3`

(F401)